Formula/mdcat.rb

```diff
@@ -0,0 +1,18 @@
+class Mdcat < Formula
+  desc "Show markdown documents on text terminals"
+  homepage "https://github.com/lunaryorn/mdcat"
+  url "https://github.com/lunaryorn/mdcat/archive/mdcat-0.6.0.tar.gz"
+  sha256 "bcc05615f6aeea1a406a481502abb12add62ac12c7f140bfe4430cda78039bb3"
+
+  depends_on "cmake" => :build
```

Comment:
I don't see anything that would need `cmake` here… other than that, the formula looks good!

Reply:
`cargo` requires `cmake` during its install.